<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainShowDialog</class>
 <widget class="QDialog" name="MainShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>656</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>视频监控系统V201412</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="show_tab_choose">
     <property name="styleSheet">
      <string notr="true">QTabWidget{display:none}</string>
     </property>
     <widget class="QWidget" name="map_show_page" native="true"/>
     <widget class="QWidget" name="video_show_page" native="true">
      <widget class="QWidget" name="widget_main" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>981</width>
         <height>581</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="aggregative_show_page" native="true">
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>981</width>
         <height>611</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="aggregative_gridLayout"/>
      </widget>
     </widget>
     <widget class="QWidget" name="show_tab_choosePage4" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
